test_unstructured/staging/test_base.py

```diff
@@ -65,13 +60,14 @@ def test_isd_to_elements():
     ]
 
 
-def test_convert_to_csv(output_csv_file):
+def test_convert_to_csv(tmp_path: str):
+    output_csv_path = os.path.join(tmp_path, "isd_data.csv")
```

Comment:
Inline the fixture this was using since this is the only user and the fixture is trivial.